src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.CodeFix.Testing/CodeFixTest`1.cs

```diff
@@ -692,7 +692,11 @@ private async Task VerifyProjectAsync(ProjectState newState, Project project, IV
 
             try
             {
-                if (expectedNumberOfIterations >= 0)
+                if (expectedNumberOfIterations == 0 && numberOfIterations <= 0)
+                {
+                    verifier.Equal(expectedNumberOfIterations, expectedNumberOfIterations - numberOfIterations, "No code fixes were expected, but a fix was offered that made no changes to the code.");
+                }
+                else if (expectedNumberOfIterations >= 0)
```

Comment:
If the previous change is taken, this can also be simplified:

```suggestion
                else if (expectedNumberOfIterations > 0)
```